tests/c_languages/CPPLintBearTest.py

```diff
@@ -7,6 +7,9 @@
 }
 """
 
+test_file2 = '\nint main() {\n' + '    int ' + \
+                         'x, ' * 100 + 'x;\n    return 0;\n}\n'
```

Comment:
weird indentation + use braces (not `\`):
```python
test_file2 = ('\nint main() {\n'
              '    int x, ' * 100 + 'x;\n' 
              '    return 0;\n'
              '}\n')
```
(I've put each line of test-code in a new line here too, so it's easier to read)

Also:
I think this one is illegal code again, because you redeclare `x` a hundred times. Redeclarations are not allowed iirc. But you can do multiple assignments:
```C
int x;
x = 3; x = 3; x = 3; x = 3; ...; ...; ......
```